operator-common/src/test/java/io/strimzi/operator/common/operator/MockCertManager.java

```diff
@@ -234,14 +344,14 @@ public void addCertToTrustStore(File certFile, String certAlias, File trustStore
 
     @Override
     public void addKeyAndCertToKeyStore(File keyFile, File certFile, String alias, File keyStoreFile, String keyStorePassword) throws IOException {
-        // never called during the tests which use this MockCertManager
-        write(keyStoreFile, "key store");
+        write(keyStoreFile, END_ENTITY_KEY_STORE);
```

Comment:
@scholzj the string was split into a constant to be consistent with the rest. 

For MockCertManager, a real Certificate is needed for the cluster operator tests to pass since the annotation generation uses the X509Certificate class. 

And since a real cert is used, it made sense to also use the real corresponding key to not lose it since it shouldn't have any impact on the tests performance.

The real key store is the one that's not needed anywhere so instead of wringing and reading a binary keystore I kept it as a string.